acoustid/future/fpindex/updater.py

```
@@ -0,0 +1,172 @@
+            SELECT id, fingerprint
+            FROM fingerprint
+            """
+        )
+        loaded_count = 0
+        async for fp in cursor:
+            await asyncio.sleep(0.1)
+            logger.info("Processing fingerprint %d", fp["id"])
+            loaded_count += 1
+            if loaded_count % 10 == 0:
+                logger.info(
+                    f"Processed {loaded_count}/{total_count} fingerprints ({loaded_count / total_count * 100:.2f}%)"
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)